static/app/components/compactSelect/list.tsx

```diff
@@ -89,22 +107,27 @@ function List<Value extends React.Key>({
   shouldFocusWrap = true,
   shouldFocusOnHover = true,
   compositeIndex = 0,
+  sizeLimit,
```

Comment:
Do we want to provide a default? We probably never want to render say 100+ items at a time?

Reply:
Yeah I thought about having a default too. Decided against it because there are certain things we need to account for before setting a limit, namely setting `searchable` to true so that hidden options can still be retrieved, and hoisting selected options to the top. Having a default limit might cause some selectors to behave badly because the developer wasn't aware of the limit and didn't do those things.